cfpb/teachers-digital-platform
# Summary and efficacy

Implemented Dimension Summaries, Efficacy survey logic (90%), and misc issues 

## Additions

- Updated the following Dimension Summaries (With calculations):
  - Utility Summary
  - Quality Summary
  - Content Summary (3: Elementary, Middle, & High)
  - Efficacy Summary (Still a work in progress)
- Implemented Efficacy logic
  - Add a new "Study" (add as many as you want)
  - Remove any "Study" (except not the first one)

## Testing

- execute ./setup.sh
- Verify the following:
- Efficacy
  - [ ] You can and and remove Efficacy "Studies" in criterion 1
- Summaries
  - [ ] Content Elementary survey answers match its summary calculation
  - [ ] Content Middle survey answers match its summary calculation
  - [ ] Content High survey answers match its summary calculation
  - [ ] Quality survey answers match its summary calculation
  - [ ] Utility survey answers match its summary calculation
  - [ ] Efficacy (criterion 2 & 3 ONLY) match its summary calculation
    - Criterion 1 (Study calculations) are still under construction


## Review

- @dcmouyard 

[Preview this PR without the whitespace changes](?w=0)
